assessments/README.md

```diff
@@ -1,18 +1,19 @@
-# Security Assessments
+# Security reviews
```

Comment:
is there a reason we distinguish between review and evaluation? Feel like these should be consistent